index.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html data-theme="dark">
```

Comment:
@olibu is this necessary?
(and it is correct when OS theme is light? or would cause flickering on the other way?)

Reply:
Thinking about it, I'd say it's fine as it is.

Reply:
I verified it in dark and light mode to work without flickering. However, I set dark as the default. When testing I recognized that the dark flash is not as disturbing as the light flash. 
Flickering stopped by separating the function into the theme.js